docs/API/statistics.md

```diff
@@ -0,0 +1,89 @@
+# Statistics
+
+All statistics provide the increase in metrics over a 12 month period in monthly intervals.
+
+## How statistics are calculated
+
+- If a month has no increase, it is skipped.
+- The date is provided as a key for that month (including a day value).
+- If a date has a day value that is **not** the final day of the month (e.g. 2022-12-15) that was the last time a value was added the interval still includes that whole month
+- _unless_ the month in question is the current month (i.e. if current date = 2022-12-16, the stats for that month are up until current day).
+
+## Endpoints
```

Comment:
I think it would also be helpful to prefix the endpoints with the appropriate HTTP verb. I'm guessing some of these endpoints expect the GET verb but I'm not sure that's the case for all of them

Reply:
Good catch, I've added HTTP values to the specific endpoints and example response